test/parallel/test-timers-unenroll-unref-interval.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const common = require('../common');
+const timers = require('timers');
+
+const keepOpen = setTimeout(() => {
+  common.fail('Test timed out.');
+}, common.platformTimeout(500)); // Keep event loop open.
```

Comment:
@Trott I'm not clear on what benefit that would have, could you elaborate?

All the unref timers are guaranteed to fire before this, it could be just 2ms theoretically in that respect, but we do need to actually keep it open long enough for the 1ms timeouts to happen twice IIRC.